### ISSUE TITLE
Redesign the way we allow users to currently filter our charts - make it more obvious this feature exists

### ISSUE THREAD

**Issue by HarryFergie** [GSS-Cogs/dd-cms#760]

**Comment by gareth-lewis** [comment]
![image.png](https://images.zenhubusercontent.com/62bc138cfba24f8002b96a0a/4209d86f-8fb3-4d39-b589-50556ece9c3d)

**Comment by JohnLewisUR** [comment]
UX team have done some design exercise to generate options. 
UR team to discuss with VAWG team and then do some pop-up testing to get feedback on the options to help determine which is more useful/helpful

**Comment by JohnLewisUR** [comment]
2 options to get feedback on. Light touch feedback on both to get an idea of which we can move forward with.

**Comment by edd-harvey** [comment]
Have we had any feedback on the preferred options for these designs @gareth-lewis @JohnLewisUR

**Comment by JohnLewisUR** [comment]
Have received some feedback on the options, and the small guidance image below the legend would be what I suggest as a first attempt - it's a relatively small change that will hopefully give the impact we need. Researched with a number of people in the office and although we had some mixed feedback about all designs, this seems the best approach to start with.
It's also worth noting that the ONS website has charts with similar functionality and they do not provide any guidance or indication on how to use the chart.
In short - quick win is to add the image and text instruction, but to review at a later date, dependant on ant future feedback, or this type of chart interactivity becoming commonplace, making the need for this additional guidance unnecessary.

**Comment by HarryFergie** [comment]
Summary of findings below:

- Legend as a term not understood by everyone (also known as a 'key') - think we need to stick with legend though as i think it is the technical term for it
- Multiple people like the tick option - seen as the most obvious ('everyone knows you can select a tick box') and least wordy option - i personally think it's to do with it having the largest block of colour too, but that's just my thinking
- Twisty dropdown is least liked as seen as being too wordy, taking up too much space on page and might not be seen or clicked (someone suggested if older people know to click those)
- Darren also had the same view as Harry, that the smallest addition would be best to start - the small icon suggesting to click the legend. But also said that he can see pros for each in different contexts
- Some concern that the icon and short text explainer might get missed, glanced over as part of the legend
- So far ticks has most votes for what people prefer, second to the short text/icon by the legend, then in last place is the twisty with text
- BUT, as mentioned yesterday the ONS site has similar functionality and doesn't have any guidance/explainer - so i half think we might not even need to do anything

Discussed with Edd and agreed to implement the small explainer as a start and then review whether this addresses the need. Can then update if required. Final design below:

**Comment by HarryFergie** [comment]
![image.png](https://images.zenhubusercontent.com/62d6af6eae090a87615b5ca8/39e6e374-36c8-4fd0-95ab-5b5ed11a1294)
